addons/website/static/src/components/dialog/seo.js

```diff
@@ -457,7 +457,10 @@ export class OptimizeSEODialog extends Component {
         }
         data.website_meta_og_img = seoContext.metaImage;
         await this.orm.write(this.object.model, [this.object.id], data, {
-            context: {lang: this.website.currentWebsite.metadata.lang},
+            context: {
+                lang: this.website.currentWebsite.metadata.lang,
+                'website_id': this.website.currentWebsite.id,
```

Comment:
Before 16.0, this was automatic (it still is for anything still attached to the `WebsiteRoot` as everything was before). We should probably check if possible to make sure this is automatic again IMO.

Reply:
I don't think we want that, it means that everything we will do from the website preview will be done with a website in the context, will that always be true?
I am confident about that before Odoo 16 since we are in the frontend but here we are in the backend, I don't know..
I thought about that and directly went with a "no go" and made this specific fix (as it's already done at some other places).

tl;dr: would you really patch `orm.call` and `orm.write` to force website_id in context everywhere when on the preview?

Reply:
I don't see what the difference is between before and after Odoo 16.0: when the preview is displayed, it is the same as before, every action should/could know you were in website-specific context when the action occurred :shrug: Which, in the case of write, would trigger the COW.

Reply:
It's also strange that it would be automatic for the editor and not the rest no?

Reply:
Maybe add an util in the website service then. Kinda like a wrapper for the orm service? :shrug: Something like `this.website.orm.XXX()` == `this.orm.XXX(/* with website id in the context */)`. In the end, patching the `orm` service directly might not be something the framework team would like, so maybe this would be better.

Reply:
What about leaving that part to https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=974&model=project.task&view_type=form&id=3518478 where I found more bugs related to this website_id context in rpc and just merge this "critical" fix already?

I am a bit lazy on that one, gotta admit, especially given the freeze rush reviews :grimacing: